packages/datatrak-web/src/Routes.tsx

```diff
@@ -4,14 +4,33 @@
  */
 
 import React from 'react';
-import { Route, Routes as RouterRoutes } from 'react-router-dom';
-import {
-  MainPageLayout,
-  SurveyPageLayout,
-  LandingPage,
-  SurveyPage,
-  SurveyQuestionsPage,
-} from './views';
+import { Navigate, Route, Routes as RouterRoutes } from 'react-router-dom';
+import { LandingPage, SurveyPage, SurveyQuestionsPage, LoginPage } from './views';
+import { useUser } from './api/queries';
+import { FullPageLoader } from '@tupaia/ui-components';
+import { ROUTES } from './constants';
+import { BackgroundPageLayout, MainPageLayout } from './layout';
+import { VerifyEmailPage } from './views/VerifyEmailPage';
+
+/**
+ * If the user is not logged in, redirect to the login page
+ */
+const LandingPageRedirect = () => {
+  const { isLoggedIn, isLoading, isFetched } = useUser();
+  if (isLoading || !isFetched) return <FullPageLoader />;
+  if (!isLoggedIn) return <Navigate to="/login" replace={true} />;
+  return <LandingPage />;
+};
```

Comment:
This needs to be more general to handle redirecting to the login page for all "private routes". [Here ](https://github.com/remix-run/react-router/blob/dev/examples/auth/src/App.tsx)is the example from the docs on that. It might be cleaner to make a PrivateRoute component or something if we have to call it in a few places. That is what we did in AdminPanel in `admin-panel/src/authentician/PrivateRoute`. Let me know if you want to pair on it?